app/assets/stylesheets/layout_components/card.scss

```diff
@@ -0,0 +1,41 @@
+.card {
+  background: $white;
+  padding: $base-padding 0;
+  text-align: center;
+  margin-bottom: $base-margin;
+
+  &__title {
+    font-size: $base-font-size*1.45;
+    padding: 7px;
```

Comment:
this actually makes the titles taller than our other section titles, it looks like the other ones use:
padding: $base-padding*0.75 $base-padding;

Reply:
oh, I took it from the membership comparisons, should we be matching this as well as our comparison radio's to be the same as our section titles?

Reply:
i think the comparison radios are a bit different - it looks like they have a line-height set on them as well which affects the padding. i think here it should match the section titles